models/graphQLSchema.js

```diff
@@ -0,0 +1,7 @@
+module.exports = (sequelize, DataTypes) => {
```

Comment:
fyi @pb82 model for GraphQLSchema. See https://github.com/aerogear/data-sync-server/pull/9/files#diff-5709bb01e7a28a79644cde19374931d6R3 for example of what gets inserted